exasol_advanced_analytics_framework/utils/udf_context_wrapper.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
```

Comment:
move pandas import to get_dataframe, because we want to avoid, if we don't need it. It takes 0.5 seconds.